src/apis/us/UserCommunicatorImpl.java

```diff
@@ -31,6 +32,13 @@ public int getUserInput() {
         System.out.println("User input retrieved");
         return num;
         */
+        int input=3;
+        
+        if(input<=0) {
```

Comment:
Parts 4 and 5 can be changed here, as I havent integrated user input beyond this varible and my construction is tied up with the CE main function. Changing this input verifies 4 and 5